emails/tests/utils_tests.py

```diff
@@ -103,3 +106,73 @@ def test_get_domains_from_settings(self):
             'RELAY_FIREFOX_DOMAIN': 'default.com',
             'MOZMAIL_DOMAIN': 'test.com'
         }
+
+@override_settings(SITE_ORIGIN='https://test.com')
+@patch('emails.utils.GENERAL_TRACKERS', ['open.tracker.com'])
+@patch('emails.utils.STRICT_TRACKERS', ['strict.tracker.com'])
+class RemoveTrackers(TestCase):
+    def setUp(self):
+        self.expected_content = (
+            '<a href="https://test.com/faq">A link</a>\n'
+            + '<img src="https://test.com/faq">An image</img>'
```

Comment:
We may need to be more clever for the next version about replacing links to images, but this does get rid of the tracker.

Reply:
Added as a follow-up item https://mozilla-hub.atlassian.net/browse/MPP-1919